hooks/pre-commit.sh

```diff
@@ -50,7 +50,7 @@ fi
 # Format all staged files, then exit with an error code if any have uncommitted
 # changes.
 echo 'Formatting staged Python files . . .'
-yapf -i -r ${PYTHON_FILES[@]}
+npm run yapf
```

Comment:
Don't do this, run `yapf` as a standalone command. There is no need for this to depend on npm or yarn its a git hook.

Reply:
I agree, but I don't think we should repeat the same command twice in `package.json` and here, since the two copies would eventually get out-of-sync and hard to maintain. I used `npm` here since the user has to have it install in order to run Origami in the first place.

Reply:
The thing is, why should need to run Origami if I just want to do some documentation changes. I don't need to have npm in that case. But still, I want my python files to be checked for linting.
Also, again we don't use npm in the first place. We use `yarn` for dependency management. These git commit hooks are not a part of our javascript code and neither should it be dependent on it.